mitodl/mit-open
# Shanbady/program letter template fields

### What are the relevant tickets?

Closes #554 



### Description (What does it do?)
This PR includes:
1. the addition of a 'ProgramLetterConfiguration' models which we use to configure program letters
2. The addition of a 'ProgramLetterSignatore' object that configures the signatures and names that go on the program letters.
3. The addition of the above models to django admin.
4. Some refactoring of utils methods so we don't get circular imports (which I had run into when making these changes - decided to go for it rather than doing some temporary workaround)

### How can this be tested?
See that ProgramLetterConfiguration  objects are configurable via django admin. Make sure tests pass. Make sure the refactor of the utils methods didnt mess anything up (there were a few references to relative paths and some tests mocked some methods that were referenced by string "learning_resources.utils...".

### Additional Context
The "meat" of the changes relevant to the ticket is in these commits: 
https://github.com/mitodl/mit-open/commit/020f071ce2139d70082fb935dca9b55eb7c647b9
https://github.com/mitodl/mit-open/commit/85583d62fd4bad43b17c564d7978be1803fd1a7f

The code refactoring is in this commit:
https://github.com/mitodl/mit-open/commit/8d10591469dcadcd43b642b3ecbbfd4fbf3bb430
